web-ui/src/main/resources/catalog/locales/en-editor.json

```
@@ -123,7 +123,11 @@
     "directory": "Folder path",
     "directoryEntries": "Entries",
     "error": "Error",
-    "errors": "Error(s)",
+    "errors": "Errors",
+    "success": "Success",
+    "successes": "Successes",
+    "warning": "Warning",
+    "warnings": "Warnings",
```

Maybe without capital letters?

```suggestion
    "errors": "errors",
    "success": "success",
    "successes": "successes",
    "warning": "warning",
    "warnings": "warnings",
```